package.json

```diff
@@ -90,7 +90,7 @@
   "homepage": "https://github.com/ReactiveX/RxJS",
   "devDependencies": {
     "benchmark": "1.0.0",
-    "benchpress": "2.0.0-alpha.37.2",
+    "benchpress": "^2.0.0-beta.1",
```

Comment:
would you able to remove `^` for dependency version? currently we're trying to have fixed version dependency with known good version and manually increment its version if update is verified.